Userland/Libraries/LibWeb/Painting/FilterPainting.cpp

```diff
@@ -0,0 +1,126 @@
+                    // Applies a linear multiplier to input image, making it appear more or less bright.
+                    // A value of 0% will create an image that is completely black. A value of 100% leaves the input unchanged.
+                    // Values of amount over 100% are allowed, providing brighter results.
+                    apply_color_filter(Gfx::BrightnessFilter { amount });
+                    break;
+                }
+                case CSS::Filter::Color::Operation::Contrast: {
+                    // Adjusts the contrast of the input. A value of 0% will create an image that is completely gray.
+                    // A value of 100% leaves the input unchanged. Values of amount over 100% are allowed, providing results with more contrast.
+                    apply_color_filter(Gfx::ContrastFilter { amount });
+                    break;
+                }
+                case CSS::Filter::Color::Operation::Invert: {
+                    // Inverts the samples in the input image. The passed parameter defines the proportion of the conversion.
+                    // A value of 100% is completely inverted. A value of 0% leaves the input unchanged.
+                    apply_color_filter(Gfx::InvertFilter { amount_clammped });
+                    break;
+                }
+                case CSS::Filter::Color::Operation::Opacity: {
+                    // Applies transparency to the samples in the input image. The passed parameter defines the proportion of the conversion.
+                    // A value of 0% is completely transparent. A value of 100% leaves the input unchanged.
+                    apply_color_filter(Gfx::OpacityFilter { amount_clammped });
+                    break;
+                }
+                case CSS::Filter::Color::Operation::Sepia: {
+                    // Converts the input image to sepia. The passed parameter defines the proportion of the conversion.
+                    // A value of 100% is completely sepia. A value of 0% leaves the input unchanged.
+                    apply_color_filter(Gfx::SepiaFilter { amount_clammped });
+                    break;
+                }
+                case CSS::Filter::Color::Operation::Saturate: {
+                    dbgln("TODO: Implement saturate() filter function!");
+                    break;
+                }
+                default:
+                    break;
+                }
+            },
+            [&](CSS::Filter::HueRotate const&) {
+                dbgln("TODO: Implement hue-rotate() filter function!");
+            },
+            [&](CSS::Filter::DropShadow const&) {
+                dbgln("TODO: Implement drop-shadow() filter function!");
+            });
+    }
+}
+
+void apply_backdrop_filter(PaintContext& context, Layout::Node const& node, Gfx::FloatRect const& backdrop_rect, BorderRadiiData const& border_radii_data, CSS::BackdropFilter const& backdrop_filter)
```

Comment:
Can we get a spec link comment for this? I think it's `https://drafts.fxtf.org/filter-effects-2/#backdrop-filter-operation`